lib/lookup.js

```diff
@@ -76,20 +76,25 @@ function resolve(context, next) {
       if (rep.stripAnsi) {
         context.module.stripAnsi = rep.stripAnsi;
       }
-      if (rep.replace) {
-        if (!rep.repo && !meta.repository) {
-          next(new Error('no-repository-field in package.json'));
-          return;
+      if (!rep.npm) {
+        if (typeof rep.replace === 'boolean' && rep.replace === false) {
+          rep.npm=true;
```

Comment:
nit: should be spaces on either side of the equality sign here, surprised that didn't fail the linter